### ISSUE TITLE
Adding World of Warcraft tootip option

### ISSUE THREAD

**Issue by Arkewas** [Skamer/Syling-Tracker#120]
Hello,
is it possible to add a option to show the WoW tooltip when hovering the mouse cursor over a quest objective, like Kaliel's tracker does ?

![Kaliel's Tracker quest tooltip](https://github.com/user-attachments/assets/68a7d7ec-88c6-4cb6-a175-69c9616af429)

**Comment by Skamer** [comment]
Yes, i don't see any problem to implement this behavior behind an option.

**Comment by Arkewas** [comment]
Thanks for considering my request.

**Comment by Skamer** [comment]
Some update.

I added a way to enable the tooltip in the 2.6.7 from the options (quests -> quest ). I plan to make an another update so the work isn't finished, only made the base implementation and its integration in the options. 

Are there information you need and you'd like to have ?

**Comment by Arkewas** [comment]
Hi,
I've updated to version 2.6.7 and the quest tooltip's works fine. If you agree I'll keep this thread opened until the work is complete. I'm just curious to know what are you planning to implement to finish the job.
Thank you very much for your work!

![Image](https://github.com/user-attachments/assets/10ff2a9b-fc82-488e-a5ec-e675fc0e3d76)

**Comment by Skamer** [comment]
Sorry for my late replay, my current plan for the next update are:
- Adding the rewards. 
- Checking if some improvements can be done for the tooltip position. 
- Checking if the tooltip can be extended for world quests and bonus tasks.

**Comment by Arkewas** [comment]
Thanks for the infos. Syling Tracker it's getting better and better and in my opinion, as I said before, currently it's the best objective tracker around.

**Comment by Skamer** [comment]
Hello,

If i encounter no unexpected issue, i would publish an update to add the rewards in the tooltip for this week end.

![Image](https://github.com/user-attachments/assets/664c7b90-ffe6-483a-85a3-4024dc96e292)
![Image](https://github.com/user-attachments/assets/6a3ba033-b09f-45b1-9f87-f64dec3f64da)
![Image](https://github.com/user-attachments/assets/72a4baa9-7827-4ba1-a338-0b1d21803228)

**Comment by Arkewas** [comment]
Hi Skamer,
nice, I can't wait.

**Comment by Arkewas** [comment]
Hi Skamer,
thanks for releasing version 2.6.10. I've enabled the rewards feature in in Syling Trackers settings "Contents - Quests - Quests" and it works perfectly. If you agree I keep this thread opened for the additional tooltip's features you mentioned a couple of weeks ago, precisely:

- Checking if some improvements can be done for the tooltip position.
- Checking if the tooltip can be extended for world quests and bonus tasks.

Keep up the good work !
